### DISCUSSION TITLE
The projects section of the site

### DISCUSSION THREAD

**Discussion by miracleonyenma** [theAlphasteam/alphasweb#4] [Ideas]
We're obviously going to add a projects section. Please read carefully and reply with any question or suggestion.
So, basically the projects section would be a gallery of previous and current projects which would also include stats about these individual projects i.e
- Name of project
- Short description
- Client
- Contributors

You can suggest to remove or add more

I don't have any resource for the UI at the moment but we need to keep it as minimal and as simple as we can.

Obviously, clicking on a project opens it up, displaying more detailed info and perhaps a cta 
Drop  suggestions

**Comment by Victory-ET** [comment]
You could also add a section for upcoming projects for those who would like to collaborate

**Reply by miracleonyenma** [reply]
Okay, that feature would be great. I guess we can organize the projects by status (i.e on-going, completed, etc.). I also think that all (open source)projects should include where collaborators can participate.
